jck/jtrunner/JavaTestRunner.java

```diff
@@ -374,19 +374,27 @@ public static boolean generateJTB() throws Exception {
 
 		if ( getJckVersionInt(jckVersionNo) >= 18 && (tests.contains("api/java_net") || tests.contains("api/java_util")) ) {
 			// Requires SHA1 enabling for jar signers in jdk-18+
-			secPropsFile = resultDir + File.separator + "security.properties";
-			System.out.println("Custom security properties to be stored in: " + secPropsFile);
 			String secPropsContents = "jdk.jar.disabledAlgorithms=MD2, MD5, RSA keySize < 1024, DSA keySize < 1024";
 			BufferedWriter bw = new BufferedWriter(new FileWriter(new File(secPropsFile)));
 			bw.write(secPropsContents);
 			bw.flush();
 			bw.close();
 		}
 		
+		if ( tests.contains("api/java_net") || tests.contains("api/java_util")) {
+			String secPropsContents = "jdk.certpath.disabledAlgorithms=MD2, MD5, SHA1 jdkCA & usage TLSServer, \\" + "\n";
+			secPropsContents += "RSA keySize < 1024, DSA keySize < 1024, EC keySize < 224,\\" + "\n";
+			secPropsContents += "include jdk.disabled.namedCurves" + "\n";
```

Comment:
This looks like zOS specific, so probably needs a if platform == zOS check... ?

Reply:
It's not z/OS specific, it affects all platforms. That z/OS issue was just a convenient place to discuss what JCK changes were needed.

Reply:
@pshipton so my confusion, is my PR presumably to fix the same Tck failure, simply enables the existing SHA1 enabling fix for jdk-11 and 17: https://github.com/adoptium/aqa-tests/pull/4059
We've not seen a need for the jdk.certpath.disabledAlgorithms and include jdk.disabled.namedCurves.
?

Reply:
While not required, this PR updates jdk.certpath.disabledAlgorithms and jdk.disabled.namedCurves to the values found in the latest security.properties, removing the SHA1 parts that cause some tests to fail.

Reply:
ah sounds good